docker/scripts/entrypoint.sh

```diff
@@ -12,7 +31,7 @@ get_arg_name_from_env_name() {
 }
 
 # Sanity check
-if [ -z "${BINARY}" ]; then
+if [ -z "${BINARY+}" ]; then
```

Comment:
This should work but the common used practice to check the var and avoid unbound var error message is
for example 
```
if [ -z "${BINARY:-}" ]; then
```
That means if BINARY is not set then set it to default empty string therefore -z option will produce `true`

However, it is up to you which way to go with. Just a comment :)

Reply:
Ok modified accordingly.